workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -109,6 +169,25 @@ public static void Launch()
             };
         }
 
+        private static string GetCommand()
+        {
+            if (Application.platform == RuntimePlatform.OSXEditor)
+            {
+                return "osascript";
```

Comment:
is this needed?

Reply:
ah just saw that this was used before as well. as it works on mac, I guess we do need it